tools/packaging/common/istio-iptables.sh

```diff
@@ -404,12 +410,6 @@ iptables -t nat -N ISTIO_OUTPUT
 # Jump to the ISTIO_OUTPUT chain from OUTPUT chain for all tcp traffic.
 iptables -t nat -A OUTPUT -p tcp -j ISTIO_OUTPUT
 
-if [ -z "${DISABLE_REDIRECTION_ON_LOCAL_LOOPBACK-}" ]; then
-  # Redirect app calls to back itself via Envoy when using the service VIP or endpoint
-  # address, e.g. appN => Envoy (client) => Envoy (server) => appN.
-  iptables -t nat -A ISTIO_OUTPUT -o lo ! -d 127.0.0.1/32 -j ISTIO_REDIRECT
-fi
-
```

Comment:
This was moved down to below the ports exclusion rules, to allow apps to call themselves on excluded ports without going through envoy. As odd as it sounds, we actually needed this to get Akka cluster bootstrapping working.

Reply:
Thank you for the explanation. I would suggest moving `OUTBOUND_PORTS_EXCLUDE` branch before this rule. Does it work?

I have another PR that moving this rule after uid rule(same as this PR) and I realized that the connection of envoyclient to envoyserver is lost.

Reply:
Thanks @silentdai for catching that. So to understand correctly, sometimes envoy makes connections back to itself. That makes sense. I think it doesn't matter how early the port exclusions are, basically when you mark a port for exclusion, you're taking ownership of it, so it can be applied before any and all redirects. I'll push an update with your suggestion.